lib/create-plugin.js

```diff
@@ -50,16 +50,18 @@ const DESTINATIONS = {
  * NOTE: query/mutation resolvers will always be captured even if returning a scalar type.
  */
 
+/* eslint sonarjs/cognitive-complexity: ["error", 57] -- TODO: https://issues.newrelic.com/browse/NEWRELIC-8797 */
```

Comment:
Yes, that's right. I think I only bumped the complexity of this function up by two points. I didn't start this fire, it was always burning since this file's been churning.